frontend-zs/src/components/stories/StoryList.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react'
+import { Container } from 'react-bootstrap'
+import Axios from 'axios';
+import './Story.scss'
+import StoryCardList from './StoryCard'
+import StoryFilter from './StoryFilter'
+
+export default function StoryList() {
+
+    const [allStories, setAllStories] = useState([])
+    const [displayStories, setDisplayStories] = useState(allStories)
+    const [displayCategories, setDisplayCategories] = useState([])
+
+    useEffect(() => {
+        Axios.get(process.env.REACT_APP_HOST + ':' + process.env.REACT_APP_PORT + '/api/stories')
+            .then(response => {
+                setAllStories(response.data)
+                setDisplayStories(response.data)
+                setDistinctCategories(response.data)
+            })
+            .catch(function (error) {
+                console.log(error.message)
+            });
+    }, [])
+
+    function setDistinctCategories(stories) {
+        let allCategories = []
```

Comment:
```suggestion
        let distinctCategories = new Set()
```
We could create the set directly and change it into a list a the end, instead going through all this conversions list->set->list